### DISCUSSION TITLE
[ZSH / MacOS 12] How do you get prompt with colors & styles like the sample screenshot?

### DISCUSSION THREAD

**Discussion by sphaxslayer** [Eugeny/tabby#5912] [Q&A] [answered]
Hi, I'm new to Tabby and wanted to give it a try. So far, it's great and provides lots of efficiency. Thanks a lot.

But I have a dumb question remaining: how / where do you configure the styles and/or color themes to have the arrow background and git branche when you are in a repository (I guess)?

![image](https://user-images.githubusercontent.com/101255246/157426198-d5e197ca-7be1-4e9b-8594-0d9c79550cf2.png)

I'm using ZSH on Mac OS 12.2.1 Monterey.

Thanks for any help

**Comment by Eugeny** [comment, marked as answer]
That's `oh-my-zsh` with the `agnoster` theme (standard): https://ohmyz.sh/

**Reply by sphaxslayer** [reply]
Oh I see... Feel like an idiot now..! Thanks a lot for your very quick answer, and thanks again for this great tool!

**Reply by xoewrl** [reply]
Can we install the 'agnoster ' theme on the client-side instead of server-side ?
To avoid config at every server.

**Comment by lawweiliang** [comment]
@sphaxslayer
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)

**Comment by sphaxslayer** [comment]
Again, my bad!

Done! :)
